src/Umbraco.Core/Logging/Serilog/Enrichers/HttpRequestNumberEnricher.cs

```diff
@@ -22,11 +24,10 @@ internal class HttpRequestNumberEnricher : ILogEventEnricher
         /// </summary>
         private const string _httpRequestNumberPropertyName = "HttpRequestNumber";
 
-        private readonly Lazy<IAppCache> _requestCache;
 
-        public HttpRequestNumberEnricher(Lazy<IAppCache> requestCache)
+        public HttpRequestNumberEnricher(Func<IFactory> factoryFunc)
```

Comment:
The issue is that we don't have IRequestCache when we need to create the logging..

I realized I could change to a `Func<IRequestCache>` instead of `Func<Factory>`..